rrc@ogarcia.me/stylesheet.css

```diff
@@ -1 +1 @@
-.no-radius.panel-corner { -panel-corner-radius: 0; }
+.remove-rounded-corners #panel .panel-corner { -panel-corner-radius: 0px; }
```

Comment:
Can you remove `px`? When the size is 0 you can omit unit.

Reply:
done